### PR TITLE
Add ActorExtractLinksHeaders

### PR DESCRIPTION
**Pull request by wdantuma**
## ActorExtractLinksHeaders

ActorExtractLinksHeaders is a ActorExtractLinks actor which adds links in link headers matching a regex pattern, usable for also following [Auxillary resource links](https://solidproject.org/TR/protocol#auxiliary-resources)  like "describedby" and "acl"

### CODE REVIEW

**Comment by rubensworks** [comment]
Thanks @wdantuma, this looks really useful! I'll have a look at it next week.

File: packages/actor-extract-links-headers/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ...
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-extract-links-headers/^1.0.0/components/context.jsonld"
+  ],
+  "actors": [
+    ...
+    {
+      "@id": "urn:comunica:default:extract-links/actors#headers",
+      "@type": "ActorExtractLinksHeaders"
+      "headersRegexes": [
+        'rel="describedby"'
```
Comment (rubensworks) [inline packages/actor-extract-links-headers/README.md:34]:
```suggestion
        "rel=\"describedby\""
```

**Comment by rubensworks** [comment]
@wdantuma I just had a look, and everything looks great to me!

The reason why the CI is still failing is because the new actor will have to be added as a dependency to `engines/query-sparql-link-traversal` and `engines/query-sparql-link-traversal-solid`: https://github.com/comunica/comunica-feature-link-traversal/blob/master/engines/query-sparql-link-traversal/package.json#L51
Could you look into adding it at those 2 locations?

**Comment by rubensworks** [comment]
Thanks @wdantuma! Released as 0.4.0.

**Comment by RubenVerborgh** [comment]
Posterior note that we should thread carefully, as this changes query semantics wrt completeness.

**Pull request merged**